apis/python/src/tiledbsoma/_dataframe.py

```diff
@@ -139,7 +139,7 @@ class DataFrame(SOMAArray, somacore.DataFrame):
         it must be ``None``.
     """
 
-    _wrapper_type = DataFrameWrapper
+    _handle_type = clib.SOMADataFrame
```

Comment:
this class seems to have lost its slots definition, e.g., it should have

```
__slots__ = ()
```

See other top-level classes as a reference